python/device/device.py

```diff
@@ -1092,6 +1128,9 @@ def main(self):
 
         logger.info("Exiting main at {}".format(datetime.datetime.utcnow()))
 
+        # 60 seconds to let app insights data flush
+        time.sleep(60)
```

Comment:
In .NET we have a way to force the AppInsights SDK to flush. No such functionality in the Python version?

If you have to wait 60 seconds, you probably should print something to the console so anyone running it locally doesn't think it is hanging.

Reply:
I'm going to go back and check.  This was a dirty fix but an explicit flush is better.